### ISSUE TITLE
Can we merge to master?

### ISSUE THREAD

**Issue by jcrowgey** [insanum/gcalcli#342]
I was interested in updating gcalcli since I saw that master hasn't been committed to in 2 years.  So, I forked it, ported to python3, updated dependencies, started a rewrite in `pythonic_snake_case` instead of `javaesqueCamelCase`, began modularizing the code: broke the package out into a cli file, a backend for interfacing with the calendar api, a utils file, a colors file, etc---set up a unit testing framework with a start of unit tests for said modularized code, set up travis-ci, and so on but then (!)  I found out that someone has been committing to this repo in a branch that's just not getting merged to master.

Well, please, check out the work I've started here: github.com/jcrowgey/gcalcli!

Really, I'd like to foster a discussion about how we can make the work that's going on more visible.  And then, in more detail, I'd love to talk about how to get some of the improvements I've been working on merged into the project.

@tresni @RedBug312 @michaelmhoffman @unit193 @yarikoptic

**Comment by insanum** [comment]
I understand the project is getting a bit stale. Some commits have been made in other branches but the project isn't moving forward as you might want. After looking at the work you did in your fork I'm open to adding you as a collaborator here. @tresni what do you think?

**Comment by tresni** [comment]
No objections from me!

-- Brian

**Comment by insanum** [comment]
Alright. @jcrowgey, you want keys to the kingdom? :-)

**Comment by jcrowgey** [comment]
Ha!  Keys the kingdom?  Sure!

But I also want to make sure the citizens (and the kings) of the kingdom are happy to have me here.  Can we talk about a roadmap?  Here are some things I'd like to do:

I'd defintely like to get development work back in master, and then release to tags (possible from a release branch).  Is there a anything that blocks merging arparse_over_gflags into master (beside, I assume, a million merge conflicts)?

After that, I'd like to get the unit testing and ci stuff that I got started on merged in.  I only added a few very rudimentary tests, but I think even just having the testing framework in place will really help.  Connected to unit tests is modularization: encapsulating things into functions and whatnot.

Aftter that, I know I'd like to change how the color stuff is handled.  I think we could simplify the class structure into some sort of dictionary.  And of course, there are other issues to address.  But those are the main topics I'm motivated to work on at the moment.
